mantidproject/mantid
# Rename Opening Height element in POLREF and CRISP IDFs

The algorithm CalculateResolution (used in reflectometry to calculated dQ/Q values) searches the IDF of the instrument associated with the InputWorkspace for a `vertical gap` tag and then uses the value along with two theta to calculate the resolution. The IDFs for POLREF and CRISP contain an element called `opening height` instead of `vertical gap` which means that CalculateResolution will not run with workspaces associated with those instruments. The objective of this PR is to rename all instances of `opening height` to `vertical gap` in the POLREF and CRISP IDFs.

**To test:**
- Make sure all unit/system/doc tests pass
- Go to `Interfaces > Reflectometry > ISIS Reflectometry (Polref)`
-  Using investigation ID 1520249 and Search Instrument `POLREF` search ICAT.
- Transfer any of the runs to the processing table
- Hit Process
- Ensure the algorithm finishes successfully.

Fixes #16024
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
